### DISCUSSION TITLE
Remove the word District from District Dropdown

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#586] [Ideas]
I can select "District" from the District dropdown when adding/editing a meeting and that should not be the case. It should be removed and certainly should not be the default choice. 

In our case, we have four districts entered as possible options and only these four should show up on the list.

![District](https://user-images.githubusercontent.com/62247421/145046562-9e4b94be-a1cd-40b1-98d0-2bc0582e3847.png)

**Comment by tech2serve** [comment]
We agree with this, and I have created an issue to get it worked on. We’ll figure out how to leave it blank if one of the admin entered districts isn’t selected.

Hard to believe this has gone for so long without being caught! :-)
